deployment/pipeline-dependencies.md

```diff
@@ -0,0 +1,73 @@
+#### Managing Dependencies
+
+In order for functions to be portable, any additional Python packages need to
+be known to the framework in order to ensure those packages are installed prior
+to running the function within a job. 
+
+In order to install packages in a cluster and make them available to any job running in the cluster, they need to be specified as part of the cluster object:
+
+```
+import graphlab as gl
+
+c = gl.deploy.hadoop_cluster.create(
+  name='hadoop-cluster',
+  dato_dist_path=<HDFS path to dato distributed deployment>,
+  additional_packages='names==0.3.0')
+
+def my_function(number = 10):
+    import names
+    people = [names.get_full_name() for i in range(number)]
+    sf = graphlab.SFrame({'names':people})
+    return sf
+
+job = graphlab.deploy.job.create(my_function, number=20, environment=c)
```

Comment:
This works but it would be nice if you can switch the order of number/environment parameter so that people would get used to pass function parameters after environment.